### ISSUE TITLE
Multiple assignment and tuple unpacking issues

### ISSUE THREAD

**Issue by rdbende** [rdbende/flake8-tkinter#3]
Looks insane, but should work

```python
import tkinter as tk
import time


root = window, btn = tk.Tk(), tk.Button(command=time.sleep(1))


btn.mainloop()
```

**Comment by insolor** [comment]
```
root[1].pack()
btn.mainloop()
```

**Comment by insolor** [comment]
Actually, the correct version is like this:
```
root = window, btn = tk.Tk(), tk.Button(command=time.sleep(1))

btn.pack()
window.mainloop()
```
I think call of `mainloop()` on widgets (like Button) must be forbidden

**Comment by rdbende** [comment]
> I think call of mainloop() on widgets (like Button) must be forbidden

Yes, this is exactly what I tried to implement, when I discovered that things like `a = b, c = 1, 2` drive the linter crazy

**Comment by Moosems** [comment]
> Looks insane, but should work
> 
> ```python
> import tkinter as tk
> import time
> 
> 
> root = window, btn = tk.Tk(), tk.Button(command=time.sleep(1))
> 
> 
> btn.mainloop()
> ```

That works?

**Comment by rdbende** [comment]
Fixed by 180c482
